packages/labs/ssr/src/lib/dom-shim.ts

```diff
@@ -46,6 +46,10 @@ export const getWindow = ({
         value,
       }));
     }
+    #shadowRoot: null | ShadowRoot = null;
```

Comment:
I'm inclined to use `private _shadowRoot` here, because AFAIK we haven't been using native class fields anywhere in our code base so far. And since we're targeting ES2020 still, the output is a little gnarly (https://www.typescriptlang.org/play?target=7#code/MYGwhgzhAEBmD29oG8CwAoa0DEAjMATtALzQB2AriCANwYZYLwAUAlCg1tAQKYAuFAmWh8AFgEsIAOjyE6maAF8MioA).

@justinfagnani WDYT about starting to use native class fields in our tools?

Reply:
Sure thing, renamed.